tests/integration/api_v3/rpc/posStakers.test.js

```diff
@@ -0,0 +1,178 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const config = require('../../../config');
+const { request } = require('../../../helpers/socketIoRpcRequest');
+
+const { invalidParamsSchema, jsonRpcEnvelopeSchema } = require('../../../schemas/rpcGenerics.schema');
+const { goodRequestSchema } = require('../../../schemas/api_v3/staker.schema');
+
+const wsRpcUrl = `${config.SERVICE_ENDPOINT}/rpc-v3`;
+
+const getStakers = async params => request(wsRpcUrl, 'get.pos.stakers', params);
+
+describe('get.pos.stakers', () => {
+	let refValidator;
+	let refValidatorAddress;
+	beforeAll(async () => {
+		do {
+			// eslint-disable-next-line no-await-in-loop
+			const response1 = await request(wsRpcUrl, 'get.transactions', { moduleCommand: 'pos:stake', limit: 1 });
+			const { data: [stakeTx] = [] } = response1.result;
+			if (stakeTx) {
+				// Destructure to refer first entry of all the sent votes within the transaction
+				const { params: { stakes: [stake] } } = stakeTx;
+				refValidatorAddress = stake.validatorAddress;
+			}
+		} while (!refValidatorAddress);
+		const response2 = await request(wsRpcUrl, 'get.pos.validators', { address: refValidatorAddress });
+		[refValidator] = response2.result.data;
+	});
+
+	it('Returns list of stakers when requested for known validator address', async () => {
+		const response = await getStakers({ address: refValidator.address });
+		expect(response).toMap(jsonRpcEnvelopeSchema);
+		const { result } = response;
+		expect(result).toMap(goodRequestSchema);
+		expect(result.data.length).toBeGreaterThanOrEqual(1);
+		expect(result.data.length).toBeLessThanOrEqual(10);
+	});
+
+	it('Returns list of stakers when requested with known validator address and offset=1', async () => {
+		const response = await getStakers({ address: refValidator.address, offset: 1 });
+		expect(response).toMap(jsonRpcEnvelopeSchema);
+		const { result } = response;
+		expect(result).toMap(goodRequestSchema);
+		expect(result.data.length).toBeGreaterThanOrEqual(1);
+		expect(result.data.length).toBeLessThanOrEqual(10);
```

Comment:
We need to update all tests cases with the condition
```suggestion
		expect(result.data.stakers.length).toBeGreaterThanOrEqual(1);
		expect(result.data.stakers.length).toBeLessThanOrEqual(10);
```